chain/cosmos/chain_node.go

```diff
@@ -830,10 +840,6 @@ func (tn *ChainNode) HasCommand(ctx context.Context, command ...string) bool {
 		return true
 	}
 
-	if err != nil {
-		return false
-	}
-
 	return true
```

Comment:
should we `return false` here?

Reply:
Yes good catch since I moved logic around, resolved 0ff41686af48eb81a4c2b57bb9152da891ffc1b7